.azure/scripts/metadata_validator.py

```diff
@@ -50,5 +54,8 @@ def validate_thumbnail(thumbnail_src):
         return False
     return True
 
-def validate_image_url(image_url):
-    return image_url.startswith(CHOREO_ACR_BASE_URL)
+def validate_image_version(image_version):
```

Comment:
The version of a Docker image does not necessarily have to follow the semantic versioning format (e.g., 1.0.1). We can skip this check.

Reply:
+1